test/cql-pytest/test_native_functions.py

```diff
@@ -72,3 +72,49 @@ def test_blobas_wrong_size(cql, table1):
     # value 0x0000007b is not a valid binary representation for type bigint".
     with pytest.raises(InvalidRequest, match='blobasbigint'):
         cql.execute(f"SELECT blobasbigint(intasblob(i)) FROM {table1} WHERE p={p}")
+
+# The mintimeuuid() function works with valid "timestamp"-type values, which
+# are 64-bit *signed* integers representing milliseconds since the epoch.
+# In particular, negative timestamps are allowed and should not be rejected.
+# The function tounixtimestamp can be used to extract the timestamp back from
+# the timeuuid. In this test we use "reasonably"-low timestamps to avoid
```

Comment:
Would be good to give some hint as to what _reasonably-low_ means in this context, e.g. by referring to `timeuuid`'s precision.